### ISSUE TITLE
Allow multiple errors to be mapped to single local error

### ISSUE THREAD

**Issue by thegranddesign** [rust-lang-nursery/error-chain#235]
The reason I sought out this crate was because I wanted to do something like this:

```
::url::ParseError => ::my_crate::errors::UrlParseError
::hyper::errors::UriError => ::my_crate::errors::UrlParseError
```

Unfortunately it doesn't seem like that's currently possible with this crate.

Am I missing something?  If not, can it be added?

**Comment by Yamakaky** [comment]
Yeah, it's not possible for now because UrlParseError must contain an enum with the two variants.

**Comment by Lymia** [comment]
You could make a variant in `errors { }`, and manually write From implementations, maybe?
